### ISSUE TITLE
YANG Zhuo HOMEWORK

### ISSUE THREAD

**Issue by YZlingyu** [BUPT-CAD/wora-app#11]
NAME: YANG Zhuo（杨卓）
PLATFORM: Win32
Data Source: Undetermined
DEADLINE: July 2nd

**Comment by huan** [comment]
No, As I told you in the class: the deadline must be a specific date. 

Please update.

**Comment by YZlingyu** [comment]
Github: [https://github.com/YZlingyu/Beer-app](https://github.com/YZlingyu/Beer-app)

**Comment by huan** [comment]
Could you please fix the ratio for your GIF video?

Because it seems wired...

**Comment by YZlingyu** [comment]
Ok, For it looks well from me. I didn't find this problem.

**Comment by YZlingyu** [comment]
I will post a new one tomorrow! Sorry for that mistake .

**Comment by YZlingyu** [comment]
![beer](https://user-images.githubusercontent.com/25507025/27515754-8a8fe2c4-59dd-11e7-947c-6534d4df943a.gif)

**Comment by YZlingyu** [comment]
Thx!

**Comment by huan** [comment]
Good job!
